server/cluster/cluster_test.go

```diff
@@ -3633,7 +3633,7 @@ func TestInterval(t *testing.T) {
 func waitAddLearner(re *require.Assertions, stream mockhbstream.HeartbeatStream, region *core.RegionInfo, storeID uint64) *core.RegionInfo {
```

Comment:
Do we need to add some tests with API mode to test forward?

Reply:
added